packages/eden/src/links/http-batch-link.ts

```diff
@@ -76,27 +76,27 @@ export function generateGetBatchRequestInformation(operations: Operation[]) {
 
   const headers = new Headers()
 
-  operations.forEach((operation, index) => {
+  for (const [index, operation] of operations.entries()) {
     let operationPath = operation.params.path ?? ''
 
     // Handle path params.
     for (const key in operation.params.options?.params) {
       const placeholder = `:${key}`
-      const param = operation.params.options.params[key as never]
-      if (param != null) {
-        operationPath = operationPath.replace(placeholder, param)
+      const parameter = operation.params.options.params[key as never]
+      if (parameter != undefined) {
```

Comment:
Should we add parameters that are explicitly `null`? There's no way to encode them and they'd appear as the string "null"...